campaign_info.toml

```diff
@@ -84,7 +84,7 @@ pagename = "B25_WMDE_iPad_DE_00_var"
 tracking = "org-pad00-250114-var"
 
 [pad.test_matrix]
-device = [ 'ipad_mini', 'ipad', 'ipad_pro_9_7_inch', 'ipad_pro_12_inch' ]
+device = [ 'ipad_mini', 'ipad_pro' ]
```

Comment:
There are 3 iPad sizes - ipad mini (8.3"), iPad (10,9") and iPad Pro/iPad Air (13"). 
See https://www.apple.com/de/ipad/compare/?modelList=ipad-pro-13-m4,ipad-10th-gen,ipad-mini-a17pro

Since one of purposes of the screenshots is discovering visual glitches at different resolutions, I'd be in favor of adding the "mid" size (named just `'ipad'`) here as well. I looked at the testingbot options - as long as we don't use iOS 18, we can use the iPad (10th Generation) as the mid-size model.

Reply:
found answers to my questions on https://testingbot.com/support/getting-started/browsers.html and created device configurations for 3 different ipad sizes